<ui version="4.0" >
 <class>RendererControllerClass</class>
 <widget class="QDialog" name="RendererControllerClass" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>236</width>
    <height>179</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Render Options</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" >
   <item>
    <widget class="QGroupBox" name="groupBox" >
     <property name="title" >
      <string>Render Options</string>
     </property>
     <layout class="QGridLayout" name="gridLayout" >
      <item row="0" column="0" >
       <widget class="QCheckBox" name="checkBox" >
        <property name="text" >
         <string>Enable Antialising</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0" >
       <widget class="QCheckBox" name="checkBox_2" >
        <property name="text" >
         <string>Enable Sky Light</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0" >
       <widget class="QLabel" name="Widht" >
        <property name="text" >
         <string>Widht</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0" >
       <widget class="QLabel" name="label_2" >
        <property name="text" >
         <string>Height</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1" >
       <widget class="QLineEdit" name="lineEdit" >
        <property name="maximumSize" >
         <size>
          <width>100</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text" >
         <string>600</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1" >
       <widget class="QLineEdit" name="lineEdit_2" >
        <property name="maximumSize" >
         <size>
          <width>100</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text" >
         <string>600</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout" >
     <item>
      <spacer name="horizontalSpacer" >
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0" >
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_2" >
       <property name="text" >
        <string>Render</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton" >
       <property name="text" >
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11" />
 <resources/>
 <connections/>
</ui>
